Doc/library/multiprocessing.shared_memory.rst

```diff
@@ -64,26 +64,34 @@ copying of data.
    memory block may be larger or equal to the size requested.  When attaching
    to an existing shared memory block, the ``size`` parameter is ignored.
 
+   *track*, when enabled, registers the shared memory block with the resource
```

Comment:
Surely there needs to be a version changed directive to clarify that `track` was added in 3.13.